lib/improver/tests/utilities/test_save.py

```diff
@@ -70,13 +84,44 @@ def test_basic(self):
         save_netcdf(self.cube, self.filepath)
         self.assertTrue(os.path.exists(self.filepath))
 
-    def test_saved_cube(self):
+    def test_cube_list(self):
```

Comment:
It would be good to add a unit test where the cubes aren't identical in terms of their variable attributes, as naively I wouldn't know what to expect when saving two cubes with an identical variable attribute, as to whether this would result in the variable attribute being promoted to be a global attribute, as it is common across both cubes. As far as I can see however, the `source_realizations` attribute does remain a variable attribute within this particular unit test.